docs/apache-airflow/deprecated-rest-api-ref.rst

```diff
@@ -31,6 +28,9 @@ available at ``/api/experimental/``.
   The experimental REST API is disabled by default. To restore these APIs while migrating to
   the stable REST API, set ``enable_experimental_api`` option in ``[api]`` section to ``True``.
 
+Airflow exposes an REST API. It is available through the webserver. Endpoints are
+available at ``/api/experimental/``.
```

Comment:
Maybe add a note that the API is named experiemental only for historical reasons? This section as a whole reads dissociative.

Reply:
Good call.